libs/spandrel_extra_arches/spandrel_extra_arches/__helper.py

```diff
@@ -27,3 +31,20 @@
     ArchSupport.from_architecture(MPRNet.MPRNetArch()),
     ArchSupport.from_architecture(MIRNet2.MIRNet2Arch()),
 )
+
+
+def install(*, ignore_duplicates: bool = False) -> list:
```

Comment:
That `list` should be typed. You just need to add `from __future__ import annotations` to make this work in py 3.8 and 3.9.

```suggestion
def install(*, ignore_duplicates: bool = False) -> list[ArchSupport]:
```